README.md

```diff
@@ -160,10 +160,10 @@ requested action. Parameters for the command will be passed inside the command v
 There are two differences with the lazy attributes:
 * First of all, for every command defined in a device, two new attributes are created in the entity with the same name
 as the command but with a prefix:
-	* `_info`: this attribute reflect the current execution status of the command. When a command request is issued by
-	the Context Broker, the IoT Agent library generates this attribute with 'PENDING' value. The value of this attribute
-	will be changed each time a command error or result is issued to the IoT Agent.
-	* `_result`: this attribute reflect the result of the execution of the defined command.
+        * `_info`: this attribute reflect the current execution status of the command. When a command request is issued by
+        the Context Broker, the IoT Agent library generates this attribute with 'PENDING' value. The value of this attribute
+        will be changed each time a command error or result is issued to the IoT Agent.
+        * `_result`: this attribute reflect the result of the execution of the defined command.
```

Comment:
The changes in this part seems to have  broken the render:

![imagen](https://user-images.githubusercontent.com/1534240/47561349-5ea4c080-d91b-11e8-9acd-5241d0c405de.png)

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/669/commits/e43c1d54fd2e857b40c6751488addc3e1462bde2